packages/terra-form/docs/Input.md

```diff
@@ -0,0 +1,16 @@
+# Input
+
+Generic input which represents an HTML input element directly.
+
+## Usage
+
+```jsx
+import React from 'react';
+import { Input } from 'terra-form';
+
+<Input
+  required={false}
```

Comment:
It was most likely when there were conflicting props, like using title as a prop to display text, when title would otherwise be spread onto the dom element and perform a different role. When the roles are the same it probably makes more sense to mirror.